lib/reverseanalyzer.cpp

```diff
@@ -174,7 +174,9 @@ struct ReverseTraversal {
                 }
                 if (!continueB)
                     break;
-                valueFlowGenericForward(assignTop->astOperand2(), analyzer, settings);
+                Analyzer::Action a = valueFlowGenericForward(assignTop->astOperand2(), analyzer, settings);
+                if (a.isInvalid())
```

Comment:
`a.isModified()` would probably be better than just `a.isInvalid()`.

Reply:
Indeed. Thanks for the review.